zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -477,25 +591,67 @@ benchmarks! {
     }: redeem_shares(RawOrigin::Signed(caller), market_id)
 
     reject_market {
+        let c in 0..63;
+        let o in 0..63;
+
         let (_, market_id) = create_market_common::<T>(
             MarketCreation::Advised,
             MarketType::Categorical(T::MaxCategories::get()),
             ScoringRule::CPMM
         )?;
+
+        let range_start = T::MinSubsidyPeriod::get();
+        let range_end = T::MaxSubsidyPeriod::get();
+        T::MarketCommons::mutate_market(&market_id, |market| {
+            market.period = MarketPeriod::Timestamp(range_start..range_end);
+            Ok(())
+        })?;
```

Comment:
This pattern seems to occur quite often. I'm getting more and more weary of convenience functions like `create_market_common`. Maybe just use `create_market`?

Reply:
17bdbafbd537de84b03a52afb011838f139e18c3